hoomd/write/gsd.py

```diff
@@ -15,28 +15,20 @@
 
 
 class GSD(Writer):
-    R""" Write simulation trajectories in the GSD format.
+    """Write simulation trajectories in the GSD format.
 
     Args:
         filename (str): File name to write.
         trigger (hoomd.trigger.Trigger): Select the timesteps to write.
-        filter_ (hoomd.filter.ParticleFilter): Select the particles
-            to write, defaults to `hoomd.filter.All`.
-        overwrite (bool): When ``True``, overwite the file. When
-            ``False`` append frames to ``filename`` if it exists and create the
-            file if it does not, defaults to ``False``.
-        truncate (bool): When ``True``, truncate the file and write a
-            new frame 0 each time this operation triggers, defaults to
-            ``False``.
-        dynamic (list[str]): Quantity categories to save in every
-            frame, defaults to property.
-        log (hoomd.logging.Logger): A ``Logger`` object for GSD
-            logging, defaults to ``None``.
-
-    .. note::
-
-        All parameters are also available as instance attributes. Only
-        *trigger* and *log* may be modified after construction.
+        filter (hoomd.filter.ParticleFilter): Select the particles to write.
+            Defaults to `hoomd.filter.All`.
+        mode (str): The file open mode. Defaults to `ab`.
```

Comment:
Probably needs to be quoted, since it's a string. Not sure if double-backticks are needed here, but I _think_ so?
```suggestion
        mode (str): The file open mode. Defaults to ``'ab'``.
```

Reply:
Yes, the double back ticks are needed.